web/static/js/components/surveys/SurveyIndex.jsx

```diff
@@ -58,13 +55,8 @@ class SurveyIndex extends Component<any, State> {
 
   initialFetch() {
```

Comment:
In this component we use `componentWillMount` but with an auxiliar function (that adds semantic). 
(Not for this PR but) As I mentioned before, we should try to be consistent across components so that reading the code gets easier.

Reply:
And... we don't need to have this initialFetch() method anyway!